src/containers/quiz/quiz-info/QuizInfo.tsx

```diff
@@ -158,4 +167,133 @@ const GradedQuizInfo = ({ points, totalPoints }: GradedQuizInfoProps) => {
   )
 }
 
-export { ActiveQuizInfo, FinishedQuizInfo, UngradedQuizInfo, GradedQuizInfo }
+type StartViewQuizInfoProps = {
+  questionsAmount: number
+  attempts: QuizAttempt
+  timeLimit: QuizTimeLimit
+  usedAttempts: FinishedAttempts
+  handleStartButton: (value: boolean) => void
+}
+const StartViewQuizInfo = ({
+  questionsAmount,
+  attempts,
+  timeLimit,
+  usedAttempts,
+  handleStartButton
+}: StartViewQuizInfoProps) => {
+  const { t } = useTranslation()
+  const [isOpen, setIsOpen] = useState(false)
+
+  const limits = {
+    isNoLimitAttempt: attempts === QuizAttempt.NoLimit,
+    isNoLimitTime: timeLimit === QuizTimeLimit.NoLimit,
+    maxAttempts: parseInt(attempts, 10) || 0
```

Comment:
Decimal number system is by default used inside `parseInt`. Also why do you need || 0?
If parseInt returns `NaN`, you should handle this situation before `parseInt` call. And when attempts can be non-number type?
```suggestion
    maxAttempts: parseInt(attempts) || 0
```

Reply:
Attempts can be non-number in case when No Limit

Reply:
You can use `isNan` for checking if parsed attempts is `NaN`

Reply:
it also works with `Number(attempts) || 0` this statement

Reply:
Yeah, I know that this code works, but code should not only just work, it should be readable and maintainable. 

That boolean check that converts to a number - is an example of code that is hard to understand, debug, and maintain (because of the `||`). Nullish coalescing operator (`??`) will not help to handle this, so I would extract parsed attempts to the separate variable somewhere at the top level of the component and use that variable here 

`maxAttempts: isNaN(parsedAttempts) ? 0 : parsedAttempts`